addons/pos_restaurant/static/src/app/split_bill_screen/split_bill_screen.xml

```diff
@@ -27,16 +27,44 @@
                             <span class="subtotal text-success">
                                 <t t-esc="env.utils.formatCurrency(newOrderPrice)" />
                             </span>
+                             <span class="total">
+                                / <t t-esc="env.utils.formatCurrency(getTotalAmount())" />
+                            </span>
                         </div>
-                        <div class="pay-button">
-                            <div class="button btn btn-lg btn-primary py-3 py-lg-5 w-100" t-on-click="createSplittedOrder">
-                                <span>Split Order</span>
+                        <t t-if="!ui.isSmall">
+                            <div class="pay-button m-3">
+                                <t t-set="nbrProduct" t-value="getProductSelectedQuantity() " />
+                                <div class="button btn btn-lg py-3 w-100"
+                                     t-on-click="createSplittedOrder"
+                                     t-att-class="nbrProduct &gt; 0 ? 'btn-primary' : 'btn-disabled'">
+                                    <i class="oi oi-chevron-right me-2" />
+                                    <span>Split Order</span>
+                                    <br/>
+                                    <span class="fs-6">
+                                        <t t-esc="nbrProduct" /> product<span t-if="nbrProduct &gt; 1">s</span> selected
+                                    </span>
+                                </div>
                             </div>
```

Comment:
These buttons aren't consistent with the buttons in mobile view of payment screen and product screen.